heroku3/helpers.py

```diff
@@ -101,39 +102,3 @@ def to_python(obj,
     # obj.__cache = in_dict
 
     return obj
-
```

Comment:
Removed the function below has it is not used anywhere (raises coverage by a little bit).